packages/api/src/hooks/useAllWalletAccounts.ts

```diff
@@ -0,0 +1,78 @@
+import { useCallback, useMemo } from 'react';
+import useCurrencyConfig from './useCurrencyConfig';
+import useWalletAccountsList from './useWalletAccountsList';
+import useAvailableWallets from './useAvailableWallets';
+
+/** A custom hook that gets the list of added and non-added wallet accounts. */
+const useAllWalletAccounts = () => {
+    const { data: non_added_wallets, ...rest } = useAvailableWallets();
+    const { data: added_wallets } = useWalletAccountsList();
+    const { getConfig } = useCurrencyConfig();
+
+    /** Get all the added and non-added wallets account */
+    const modified_wallets = useMemo(() => {
```

Comment:
consistency - we shouldn't generally mix naming conventions like modified_wallets and sortWalletsByCurrency withing one file (ideally we should have consistent strategy across codebase, but if that not viable at the moment, at least we shouldn't mix them up)

Reply:
do we have any agreement whats the agreed convention for variable naming @markwylde-deriv @yashim-deriv @nijil-deriv for that?

Reply:
We should be following airbnb's style guide.